test/plugins/features/movies/index.test.js

```diff
@@ -45,4 +48,31 @@ describe('movies integration', () => {
 
   });
 
+  describe('add location', () => {
+
+    it('adds a location to a movie', async () => {
+      const response = await Movies.inject({
+        url: `/movies/${testMovie.id}/locations`,
+        method: 'POST',
+        payload: { locationName: testLocation.name }
+      });
+
+      expect(response.statusCode).to.eql(200);
+      expect(response.result.object).to.eql('movie');
+      expect(response.result.locations.length).to.eql(1);
+      expect(response.result.locations.models[0].get('name')).to.eql(testLocation.name);
+    });
+
+    it('attempts to add a location to a non-existent movie', async () => {
```

Comment:
Typically for these test names, try to focus on the expected behavior. So instead of `attempts to add`, this test should probably be `rejects...` or `404s when...`